uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_leave.scss

```diff
@@ -1,16 +1,18 @@
+$leave-input-height: 40px;
```

Comment:
Can we use a more specific name for this variable? Not all inputs are affected by this style, but only some of them. Here are some suggestions: 
- `$leave-emphasised-input-height`
- `$leave-bigger-input-height`
- `$leave-taller-input-height`